internal/cache/file/downloader/job.go

```diff
@@ -463,3 +469,41 @@ func (job *Job) GetStatus() JobStatus {
 	defer job.mu.Unlock()
 	return job.status
 }
+
+func (job *Job) validateChecksum() (err error) {
+	if !job.enableCrcCheck {
```

Comment:
Since this will be called from multiple places -> sequential & parallel download i would prefer putting this check inside this method.

Rename id done.